bn-models/src/models/ashop/ashop-cruise.ts

```diff
@@ -1,12 +1,21 @@
 // A-SHOP Cruise
 import { Base } from '../_base';
 import { PersonAlias, Vessel } from '../_lookups/index';
-import { BoatnetDate, CouchID, NonFishingDay } from '../_common/index';
+import { BoatnetDate, CouchID, LocationEvent } from '../_common/index';
 
 export const AshopCruiseTypeName = 'ashop-cruise';
 
 declare type AshopContact = PersonAlias;
 
+interface NonFishingDay {
+  date?: BoatnetDate;
+  reason?: string;
+  notes?: string;
+  location?: LocationEvent;
+  tripNum?: number;
+  tripId?: string;
```

Comment:
I think this should be ```tripId?: CouchID;```

Reply:
oh yes that's correct i'll make the change